apps/admin/package.json

```diff
@@ -46,10 +46,11 @@
     "@types/react-dom": "^18.2.7",
     "@types/react-syntax-highlighter": "^15.5.7",
     "autoprefixer": "^10.4.14",
+    "dnd-core": "^16.0.1",
```

Comment:
`dnd-core` is one of the deps that was being flagged by [`import/no-extraneous-dependencies`](https://github.com/import-js/eslint-plugin-import/blob/v2.31.0/docs/rules/no-extraneous-dependencies.md). It was already being installed as a transitive dependency. Now it's more directly version-pinned.

Reply:
oh interesting i thought pnpm stopped transitive deps!

Reply:
Maybe it does? 🤷 I'm not super confident in my pnpm-fu.

Another strategy could be to just disable `import/no-extraneous-dependencies`.